index.js

```diff
@@ -47,8 +47,32 @@ const isObject = value => value !== null && typeof value === 'object';
 const supportsAbortController = typeof globals.AbortController === 'function';
 const supportsStreams = typeof globals.ReadableStream === 'function';
 
+const mergeHeaders = (...sources) => {
+	const result = {};
+
+	for (const source of sources) {
+		if (!isObject(source)) {
+			throw new TypeError('The `headers` argument must be an object');
+		}
+
+		const headers = new globals.Headers(source);
+
+		for (const [key, value] of headers) {
```

Comment:
`of Object.entries(headers)`

Reply:
[Wait, well done.](https://github.com/sindresorhus/ky/pull/235/files#r382930813)